pkg/activator/util/header.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
```

Comment:
Should this live in the toplevel `pkg` if its used across components?

Reply:
Actually - `GetLastHeaderValue` could be in a top level package

Reply:
Ah, you're right. I guess it's fine to leave them all where they are then, even if in theory they could be broadened up.

Reply:
Moved it to `pkg/http`